docs/contributor_setup.rst

```diff
@@ -23,7 +23,7 @@ Now you can make your changes locally.
 5. When you're done making changes, check that your changes pass the tests and flake8::
 
     $ flake8 cookiecutter tests
```

Comment:
We should change this to use tox too, since the `flake8` dependency won't be installed by `python setup.py develop`